optaweb-vehicle-routing-backend/src/test/resources/mockito-extensions/README.adoc

```diff
@@ -0,0 +1,6 @@
+The Mockito extension file link:org.mockito.plugins.MockMaker[org.mockito.plugins.MockMaker]
+enables
+https://static.javadoc.io/org.mockito/mockito-core/2.27.0/org/mockito/Mockito.html#39[
+Mocking final types (Mockito 2.27.0 API)].
```

Comment:
Avoid hardcoding versions in docs (or anything outside pom/package files), they tend to go stale easily.
Use this URL instead: https://javadoc.io/page/org.mockito/mockito-core/latest/org/mockito/Mockito.html